spec/rails_helper.rb

```diff
@@ -170,12 +170,6 @@ def self.capybara_tmp_path
     Capybara.server = :puma, { Silent: true }
   end
 
-  config.before(:each) do
-    if ENV['EVENTS_READ'] == 'true'
-      allow(Event).to receive(:read_events?).and_return(true)
```

Comment:
I don't think we can do this - even though we're setting it to true in the seeds, it won't be set to true in the actual tests. If we want to always run it with true, we should just remove lines 174 and 176 and leave the rest in.

Reply:
Oh right ... seeds lot loaded in specs. I forgot. OK -- I'm putting that back. I think another week or two and we should remove all of the remaining feature-flag related things for Events, but we can do that separately.